src/wallet/rpc/wallet.cpp

```diff
@@ -132,6 +133,9 @@ static RPCHelpMan getwalletinfo()
     obj.pushKV("descriptors", pwallet->IsWalletFlagSet(WALLET_FLAG_DESCRIPTORS));
     obj.pushKV("external_signer", pwallet->IsWalletFlagSet(WALLET_FLAG_EXTERNAL_SIGNER));
     obj.pushKV("blank", pwallet->IsWalletFlagSet(WALLET_FLAG_BLANK_WALLET));
+    if (int64_t birthtime = pwallet->GetBirthTime(); birthtime != std::numeric_limits<int64_t>::max()) {
```

Comment:
Shouldn't this be `NO_KEY_TIME` explicitly?

Reply:
yeah, it could be `NO_KEY_TIME` or could also return an optional.

The issue that I see with the `NO_KEY_TIME` constant is where to place it. Because, if we place it at the wallet.h level, we would add a circular dependency wallet -> spkm -> wallet.

Reply:
Agree that this should be `NO_KEY_TIME`. Since wallet already includes spkm, leaving it there will not be circular, and it can still be accessed by things that include wallet.

Reply:
Done. Updated.